### PR TITLE
Add Devdogs v0.2.1

### LINKED ISSUE #4: Release on brew (cask)

### PR DESCRIPTION
**Pull request by mendelk**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

resolves ragingwind/devdogs#4

### CODE REVIEW

File: Casks/devdogs.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'devdogs' do
+  version '0.2.1'
+  sha256 '004cb15be88bbef2516d0ed3d75f6e4e198b5a0700ecfeba4131f34cc1cb869f'
+
+  url "https://github.com/ragingwind/devdogs/releases/download/v#{version}/Devdogs-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/devdogs.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/devdogs.rb (hunk 1 above)
Comment (mendelk) [inline Casks/devdogs.rb]: Done!

File: Casks/devdogs.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
   sha256 '004cb15be88bbef2516d0ed3d75f6e4e198b5a0700ecfeba4131f34cc1cb869f'
 
   url "https://github.com/ragingwind/devdogs/releases/download/v#{version}/Devdogs-#{version}.zip"
+  appcast 'https://github.com/ragingwind/devdogs/releases.atom'
```
Comment (vitorgalvao) [inline Casks/devdogs.rb]: You also need the `checkpoint`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/devdogs.rb (hunk 2 above)
Comment (mendelk) [inline Casks/devdogs.rb]: It should be there. Refresh maybe?

**Review by vitorgalvao — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**